wrangler/src/txFunctions/run.js

```diff
@@ -218,12 +137,10 @@ export default async ({ request, params, env, ctx }) => {
     xdr,
     signer: txFunctionSignerPublicKey,
     signature: txFunctionSignature,
-    cost,
+    cost: cost,
+    feeSponsor: feeSponsor,
+    feeBalanceRemaining: feeBalanceRemaining,
   }, {
     stopwatch: watch,
-    headers: {
-      'X-Fee-Total': feeTotal,
-      'X-Fee-Spent': feeSpent
-    }
   })
 }
```

Comment:
So many changes 😱 

Looks good to me but would be nice to get more eyes here

cc: @orbitlens

Reply:
I have a knack for ending up with diffs way more complicated than the changes actually are...